### PR TITLE
Add more info at stdci teardown

### PR DESCRIPTION
**Pull request by qinqon**
It will list k8s pods and events before cluster-down

### CODE REVIEW

File: automation/test.sh [hunk 1]
```diff
@@ -2,10 +2,16 @@
 
 kubectl() { cluster/kubectl.sh "$@"; }
 
+teardown() {
+    kubectl get event || true
+    kubectl get pod || true
```
Comment (phoracek) [inline automation/test.sh]: don't we need to specify namespace?

File: automation/test.sh [hunk 2]
```diff
@@ -2,10 +2,16 @@
 
 kubectl() { cluster/kubectl.sh "$@"; }
 
+teardown() {
+    kubectl get --all-namespaces event || true
+    kubectl get --all-namespaces pod || true
```
Comment (phoracek) [inline automation/test.sh]: this would be madness on openshift, too many pods. are we interested in anything but nmstate namespace?

Comment (qinqon) [reply automation/test.sh]: Let's have it all, you never know what you need and this is just after finishing it.

Comment (phoracek) [reply automation/test.sh]: (*demonic voice* you will regret it JAJAJA)

ack

**Pull request merged**